### PR TITLE
Update node-version-manager extension

### PR DESCRIPTION
**Pull request by LeOndaz**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by LeOndaz** [comment]
image

but failing on ci

File: extensions/node-version-manager/src/utils/managers/nvm.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ const nvmUninstall = (version: string) => {
   return output;
 };
 
-const nvmDefault = (version: string) => {
+const nvmDefault = async (version: string) => {
```
Comment (AndresMorelos) [inline extensions/node-version-manager/src/utils/managers/nvm.ts:103]: Remove Async

File: extensions/node-version-manager/src/utils/shell.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const run = <T extends object | string>(fullCommand: string, parseOutput?: (op: string) => T): T => {
+  const [command, ...args] = fullCommand.split(' ');
+  const { stdout, stderr } = spawnSync(command, args, {
+    env: env(),
+    shell: '/bin/zsh',
+  });
+
+  if (stderr.length > 0) {
+    const error = stderr.toString();
+    console.error(error);
```
Comment (AndresMorelos) [inline extensions/node-version-manager/src/utils/shell.ts:13]: remove console.error, the throw new Error need to be catched and show a Toast

File: extensions/node-version-manager/src/utils/shell.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { spawnSync } from 'node:child_process';
+import { env } from './common';
+
+const run = <T extends object | string>(fullCommand: string, parseOutput?: (op: string) => T): T => {
```
Comment (AndresMorelos) [inline extensions/node-version-manager/src/utils/shell.ts:4]: Please update nvm manager to use this util, just to not duplicate code.

**Review by AndresMorelos — CHANGES_REQUESTED** [review]
Some changes requested

File: extensions/node-version-manager/src/utils/shell.ts (hunk 3 above)
Comment (LeOndaz) [reply extensions/node-version-manager/src/utils/shell.ts:4]: Hi, sorry I prefer not to touch existing code as I have no time to refactor

Comment (AndresMorelos) [reply extensions/node-version-manager/src/utils/shell.ts:4]: Ok, I think I can accept it, and then try to refactor it.

**Review by AndresMorelos — APPROVED** [review]
LGTM

File: extensions/node-version-manager/src/utils/shell.ts (hunk 3 above)
Comment (LeOndaz) [reply extensions/node-version-manager/src/utils/shell.ts:4]: thanks

**Comment by LeOndaz** [comment]
@AndresMorelos how and when do we merge?

**Comment by AndresMorelos** [comment]
@LeOndaz we need to wait for someone from raycast team

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**